src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -59,8 +59,10 @@ class PdlClient(
                 }
                 null
             } else {
-                COUNT_CALL_PDL_GT_SUCCESS.increment()
-                pdlPersonReponse.data
+                pdlPersonReponse.data?.let { pdlHentGeografiskTilknytning ->
```

Comment:
Hvis du har veldig lyst, kan du kanskje lage en PR for å endre denne fra `Reponse` til `Response` også? 😃

Reply:
Jepp, jeg ordner det.